documentation/content/Supported Plugins/multi-part-plugin.md

```diff
@@ -11,6 +11,9 @@ Keywords:
 Authors: Talha Mansoor, Jack De Winter
 ---
 
+!!! important "Deprecation"
+The `multi-part` plugin has been deprecated in favor of the `series` plugin. For an up-to-date article on using the `series` plugin, [go here]({filename}../Supported Plugins/series-plugin.md).
```

Comment:
You don;'t need `../`

Compare

`{filename}../Supported Plugins/series-plugin.md)`
`{filename}./series-plugin.md)`

Use the second one.

Reply:
Done.  (In my own blog, I try and keep all articles with full paths to make it easier to scan.)